tests/smoke.spec.ts

```diff
@@ -406,6 +406,179 @@ test.describe("OSDEV-1230: Smoke: Facilities. Upload a list in CSV format.", ()
   });
 });
 
+test("OSDEV-1234: Smoke: Create Embedded Map with no facilities on it.", async () => {
+  const browser = await chromium.launch({ headless: true }); // set headless: false to run with UI
+  const context = await browser.newContext();
+
+  // Reset cookies and storage
+  await context.clearCookies();
+  await context.clearPermissions();
+
+  // Open the admin page
+  const adminPage = await context.newPage();
+  // 1. Check your user in the admin panel
+  const { BASE_URL } = process.env;
+  await adminPage.goto(`${BASE_URL}/admin/api/contributor/`!);
+
+  // make sure that we are on the login page of Admin Dashboard
+  const title = await adminPage.title();
+  expect(title).toBe("Log in | Django site admin");
+  await expect(adminPage.getByText("Open Supply Hub Admin")).toBeVisible();
+
+  // fill in login credentials
+  const { USER_ADMIN_EMAIL, USER_ADMIN_PASSWORD } = process.env;
+  await adminPage.getByLabel("Email").fill(USER_ADMIN_EMAIL!);
+  await adminPage.getByLabel("Password").fill(USER_ADMIN_PASSWORD!);
+  await adminPage.getByRole("button", { name: "Log In" }).click();
+  await expect(adminPage.getByText(`Welcome, ${USER_ADMIN_EMAIL}`)).toBeVisible();
+
+  // make sure that we have successfully logged in
+  await expect(
+    adminPage.getByRole("link", { name: "Open Supply Hub Admin" })
+  ).toBeVisible();
+  await expect(adminPage.getByText("Select contributor to change")).toBeVisible();
+  const searchInput = adminPage.getByRole("textbox", { name: "Search" });
+  await searchInput.fill(USER_ADMIN_EMAIL!);
+  await adminPage.getByRole("button", { name: "Search" }).click();
+  await adminPage.waitForLoadState("networkidle");
+
+  const firstRowLink = adminPage.locator("table#result_list tbody tr").first().locator("th.field-__str__ a");
+  await firstRowLink.click();
+  await adminPage.waitForLoadState("networkidle");
+
+  await expect(adminPage.getByText("Change contributor")).toBeVisible();
+  const adminInput = adminPage.locator("#id_admin");
+  expect(await adminInput.locator("option:checked").textContent()).toBe(USER_ADMIN_EMAIL);
```

Comment:
Can we move this `await` to the begging of the line? Looks like it won't change the lgoic.